ConfigurationSystem/Client/Helpers/Registry.py

```diff
@@ -414,11 +402,15 @@ def getDNProperty(userDN, value):
       :return: S_OK(basestring,list)/S_ERROR() -- basestring or list that contain option value
   """
   result = getUsernameForDN(userDN)
-  if not result['OK']:
-    return result
-  secDN = __getDNSectionName(userDN)
-  return S_OK(gConfig.getValue("%s/Users/%s/DNProperties/%s/%s" %
-                               (gBaseRegistrySection, result['Value'], secDN, value)))
+  if result['OK']:
+    result = gConfig.getSections("%s/Users/%s/DNProperties" % (gBaseRegistrySection, result['Value']))
+    if result['OK']:
+      for section in "%s/Users/%s/DNProperties/%s" % (gBaseRegistrySection, user, result['Value']):
+        if userDN == gConfig.getValue("%s/DN" % section):
+          return S_OK(gConfig.getValue("%s/%s" % (section, value)))
+
+  return S_ERROR('No properties found for %s%s' %
```

Comment:
please avoid changing the error. If somethign returns S_ERROR, just propagate it

Reply:
I think in this specific case it makes sense to return `S_ERROR` with an explanation of the error.

Reply:
...at the same time I think that this code was more clearly written before...

Reply:
we lose the error number and the stacktrace

Reply:
I made this part the same as in other `Registry` methods